ComposeAdvanced/app/src/main/java/com/example/android/wearable/composeadvanced/presentation/ui/landing/LandingScreen.kt

```diff
@@ -67,68 +67,74 @@ fun LandingScreen(
     Box(modifier = Modifier.fillMaxSize()) {
 
         // Places both Chips (button and toggle) in the middle of the screen.
-        Column(
-            modifier = Modifier
-                .fillMaxSize()
-                .padding(horizontal = 10.dp),
-            verticalArrangement = Arrangement.spacedBy(4.dp, Alignment.CenterVertically)
+        ScalingLazyColumn(
+            modifier = Modifier.fillMaxSize(),
+            state = ScalingLazyListState(initialCenterItemIndex = 1)
```

Comment:
These are defaults - we shouldn't need them and so I recommend we remove

Reply:
Woah - fillMaxSize() is a default?